assets/dev/js/editor/controls/media.js

```diff
@@ -243,6 +252,54 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 	onBeforeDestroy() {
 		this.$el.remove();
 	},
+
+	async onMediaImageSize() {
+		this.ui.mediaImageSize.css( 'color', 'inherit' );
+		const previewPlaceHolder = this.ui.mediaImage;
+		const selectedImage = {
+			id: this.getControlValue( 'id' ) || previewPlaceHolder[ 0 ].getAttribute( 'data-id' ),
+			alt: this.getControlValue( 'alt' ) || previewPlaceHolder[ 0 ].getAttribute( 'alt' ),
+		};
+
+		const selectedSize = this.ui.mediaImageSize.val();
+		let imageURL = await elementor.imagesManager.getImageUrl( {
+			id: selectedImage.id,
+			size: selectedSize,
+		} );
+		const stateOptions = {
+			url: null,
+			id: selectedImage.id,
+			alt: selectedImage.alt,
+			dimensions: selectedSize,
+			source: 'library',
+		};
+		if ( imageURL ) {
+			stateOptions.url = imageURL;
+			this.setValue( stateOptions );
+		} else {
+			elementor.channels.editor.once( 'imagesManager:detailsReceived', ( data ) => {
+				imageURL = data[ selectedImage.id ][ selectedSize ];
+				if ( imageURL ) {
+					stateOptions.url = imageURL;
+					this.setValue( stateOptions );
+				}
+			} );
+		}
+	},
+
+	toggleSizeControl() {
+		const previewImageExist = this.$el.find( '.elementor-control-media__preview' )[ 0 ]?.style?.backgroundImage;
```

Comment:
you have this under `this.ui.mediaImage`

Reply:
Using `this.ui.mediaImage`